science/rtl-sdr/Portfile

```diff
@@ -6,13 +6,13 @@ PortGroup           github 1.0
 
 name                rtl-sdr
 
-github.setup        osmocom rtl-sdr 5e73f90f1d85d8db2e583f3dbf1cff052d71d59b
-version             20220115-[string range ${github.version} 0 7]
-checksums           rmd160  b20663bafc564401df066039a4e83a5215d99093 \
-                    sha256  19ed112e1f7f425dd77c98d81d205ebe8ec7aa372e6a92bcb2a50cf6eca274fd \
-                    size    134207
+github.setup        osmocom rtl-sdr 2.0.2 v
+version             ${github.version}
```

Comment:
`version` field is set automatically from the `github.setup` line; please remove